calico/getting-started/kubernetes/self-managed-onprem/onpremises.mdx

```diff
@@ -174,7 +191,14 @@ The geeky details of what you get:
 
 :::note
 
-The **etcd** database is not recommended for new installs. However, it is an option if you are running {{prodname}} as the network plugin for both OpenStack and Kubernetes.
+The **etcd** datastore is not recommended for new Kubernetes installs:
+
+* etcd is another component to manage and maintain.
+* Some newer Kubernetes-targeted features (such as Service matches in policy) are not supported with the etcd datastore.
+* The Cloud/Enterprise versions of {{prodname}} do not support etcd mode at all so using this manifest prevents
+  upgrade to those versions of {{prodname}}.
+
+However, it is an option if you are running {{prodname}} as the network plugin for both OpenStack and Kubernetes.
```

Comment:
Do you want to add: "Typha is not required or supported in etcd mode" ?

Reply:
It's already mentioned further up